IPV4_Addressing/README.md

```diff
@@ -0,0 +1,357 @@
+- Under classful addressing, an organization with, say, 2,000 hosts was typically allocated a class B (/16) subnet address- —leaving more than 63,000 addresses that could not be used by other organizations. <br/>
+- This led to a rapid depletion of the class B address space and poor utilization of the assigned address space.  <br/>
+
+<br/> 
+How could we improve this? For this purpose, from classful adressing, there was a transition to CIDR i.e Classless Interdomain Routing Protocol
+<br/><br/>
+
+## ACTIVITY 4:
+To reduce the wastage of addresses(as seen in classful addressing) we use CIDR. This is also known as Supernetting
+<br/>
+
+### ACTIVITY 4.1:
+As per Activity 2, we would now comfortable with identifying network addresses. <br/>
+So, given the network address , will you be able to identify the first and last host in the network?<br/>
+How would you do this?<br/>
+Think!<br/>
+HINT: Calculate the number of hosts that can be supported in the network.<br/>
+
+[Need Help?](Help.txt)
+
+<br/><br/><hr>
+
+### ACTIVITY 4.2:
+
+
+Representation for CIDR: <br/>
+**a.b.c.d /x** <br/>
+x represents the network/subnet mask. <br/><br/>
+
+- The x most significant bits of an address of the form a.b.c.d/x constitute the network portion of the IP address, and is often referred to as the prefix (or network prefix) of the address. <br/>
+
+- The remaining 32-x bits of an address can be thought of as distinguishing among the devices within the organization, all of which have the same network prefix. <br/><br/>
+
+For example ,consider the CIDRized address a.b.c.d/21 <br/>
+
+The first 21 bits of the CIDRized address a.b.c.d/21 specify the organization’s network prefix and are common to the IP addresses of all devices in that organization. <br/>
+The remaining 11 (32-21) bits then identify the specific hosts in the organization.
+<br/><br/>
+
+Take [Quiz4](Quiz/Quiz4/Quiz4.txt)  to strengthen your knowledge about this section ! <br/>
+You can submit the solution [here](Solutions/Quiz4/Solutions4.txt).
+
+<br/>
+
+<hr>
+
+### ACTIVITY 4.3: 
+
+Refer to [this](Tricks.txt) file for some tricks to identify Network id, Broadcast id, First IP and Last IP
+<br/>
+
+Take [Quiz5](Quiz/Quiz5/Quiz5.txt) to test your understanding now ! <br/>
+You can submit the solution [here](Solutions/Quiz5/Solutions5.txt).
+
+
+<hr>
+
+Now that youve completed all the quizes, let 's see how you've fared. Hopefully, you have noted down your scores.
+<br/>
+<br/>
+
+You know what, it does not matter how much you've scored, what matters is how much you've learned!<br/>
+So,never stop learning ! ;)
+<br/>
+
+### FURTHER EXPLORATION:
+All this is just the tip of the iceberg. As the field of Computer Networks is very vast, there is quite a lot that can be learned. You could learn about 
+1. IPV6 addressing <br/>
+2. MAC addresses <br/>
+3. Dive deep into TCP/IP stack <br/>
+4. Learn about OSI stack and difference with TCP/IP stack <br/>
+and many many more !
+
```

Comment:
@Anjalikamath I really appreciate your efforts. This MicroByte has some nice scope for learners. Just one request though. Can you kindly add a few references in the 4 points mentioned in the `FURTHER EXPLORATION` section.

Other than that, it LGTM! :rocket: 

Let's see what the other reviewers have to say.

Reply:
I have modified the file.